.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Question
+about: Ask a question about RMG
+
+---
+
+<!-- Thanks for your interest in RMG! Please first check the online documentation at reactionmechanismgenerator.github.io/RMG-Py to see if it answers your question. Also, try searching GitHub issues to see if someone has asked your question before. -->
```

Comment:
Will the documentation url become a link?

Reply:
I don't think so, since this is just added as a comment. It won't show up once they submit the issue. To go there, they'll have to copy the link, which is not optimal.